src/runtime_src/doc/toc/p2p.rst

```diff
@@ -20,7 +20,7 @@ BIOS Setup
 
 Note
 .......
-It may be necessary to update to the latest BIOS release before enabling P2P.  Not doing so may cause the system to continuously reboot during the boot process.  If this occurs, power-cycle the system to disable p2p and allow the system to boot normally.
+It may be necessary to update to the latest BIOS release before enabling P2P.  Not doing so may cause the system to continuously reboot during the boot process.  If this occurs, power-cycle the system to disable P2P and allow the system to boot normally.
```

Comment:
From my own experience, updating to the latest BIOS is good to use any card, whatever its use...